ethanschreur/kids-krafts
# Customers can contact the owners with a contact us form

**What does this PR do?**
- This pull request allows customers to fill out a contact form to send an email to the sellers

**Description of the task to be completed** 
- Add the contact us form page
- Use flask mail to send the email
- Make an email HTML template that gets sent
- Upon the successful sending of an email, flash a success message

**How should this  be tested**
- (On Windows) Run `. venv/scripts/activate` 
- `python run.py`
- Go to http://127.0.0.1:5000/contact
- Enter the contact form using your email
- Submit the form
- Check that the form is cleared and a success message is displayed on the page
- Check your email for the email that should have been sent to you.
